examples/hello-world/step-by-step/higgs/sklearn-linear/code/sgd_fl.py

```diff
@@ -139,25 +139,22 @@ def main():
                 model.intercept_ = global_params["intercept"]
 
         # (6) evaluate global model first.
-        global_accuracy, global_report = evaluate_model(x_test, model, y_test)
+        global_auc, global_report = evaluate_model(x_test, model, y_test)
         # Print the results
-        if curr_round % 10 == 0:
-            print(f"global model Accuracy: {global_accuracy:.2f}")
-            print("global model Classification Report:\n", global_report)
+        print(f"{site_name}: global model AUC: {global_auc:.4f}")
```

Comment:
I removed the report print to avoid too many lines, and the number does not change much after 10 rounds. Since we currently only run for 30 rounds, I think it should be fine.